internal/storage/secrets.go

```diff
@@ -84,7 +82,22 @@ func GetToken(name string, ss SecretStore) (Token, error) {
 		}
 	}
 
-	return Token{}, ErrTokenNotFound
+	return Token{}, nil
+}
+
+func TokenExists(name string, ss SecretStore) (bool, error) {
+	secrets, err := ss.Get()
+	if err != nil {
+		return false, err
+	}
+
+	for _, token := range secrets.Tokens {
```

Comment:
this is fine but makes me think `Tokens` should just be a `map[string]Token` instead of `[]Token`

Also maybe this would make sense as a method on `SecretStore`?

Reply:
Yeah, as I was going through this there were a bunch of things that seemed like they could be reorganized. I want to get the fix out and then loop back.